src/routes/Send.tsx

```diff
@@ -451,11 +451,15 @@ export function Send() {
     function processInvoice(source: ParsedParams & { invoice: string }) {
         state.mutiny_wallet
             ?.decode_invoice(source.invoice!)
-            .then((invoice) => {
+            .then(async (invoice) => {
                 if (invoice.expire <= Date.now() / 1000) {
                     navigate("/search");
                     throw new Error(i18n.t("send.error_expired"));
                 }
+                const isHodl = await MutinyWallet.is_potential_hodl_invoice(
```

Comment:
I think this should just happen below in a single spot where it is being set. Now you have it here twice. I think this method can just replace the previous.